watcher/watcher.go

```diff
@@ -268,14 +274,17 @@ func (w *Watcher) reportRoomsStatusesRoutine() {
 }
 
 func (w *Watcher) watchRooms() error {
-	w.RemoveDeadRooms()
-	w.AutoScale()
-	w.AddUtilizationMetricsToRedis()
+	l := w.Logger.WithFields(logrus.Fields{
+		"operation": "watcher.watchRooms",
+	})
+	w.WithRedisLock(l, w.AutoScale)
```

Comment:
Why do we need full redis lock here?

Reply:
I just kept this and AddUtilizationMetricsToRedis as it was. I'm going to think about using downScaling

Reply:
ah okay! sorry, with the diff I didn't notice this was already AddUtilizationMetricsToRedis and AutoScale behaviour to lock everything